diagnostic.md

```diff
@@ -7,19 +7,19 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:acballard/git-basics-diagnostic.git. -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!--->git commit.
```

Comment:
If you commit after cloning, there will be no changes to commit.

Answer: create a new working branch.

you want to create a new branch to work on, and add and commit changes as you make them.